### ISSUE TITLE
user-central configuration defining experimenter, institution, lab, ... defaults

### ISSUE THREAD

**Issue by yarikoptic** [NeurodataWithoutBorders/pynwb#1230]
An idea came up while watching @rly 's tutorial.  As those fields are optional, they could often be forgotten to be entered etc. And in many (not all) cases some or all of them would be common for many (if not all) produced .nwb files   .  So I thought, why similar to `git` having a configuration with user.name and user.email there couldn't be some `~/.nwbconfig` file with section `defaults` providing default values for those (and possibly other?) fields?

**Comment by oruebel** [comment]
I agree that this could be a very useful feature.  I believe @bendichter is doing something like this in his conversion tools  https://github.com/catalystneuro/nwb-conversion-tools

**Comment by yarikoptic** [comment]
yes, it could be implemented in the tools above pynwb, and be ad-hoc to them. It is just because ATM majority of interaction with pynwb is happening directly at the user level (all the tutorials for data conversion etc use pynwb directly), I thought it might be worthwhile generally at pynwb level.

**Comment by oruebel** [comment]
> I thought it might be worthwhile generally at pynwb level.

I do agree with you that this is something that would be useful at the PyNWB level
